src/tabs/tabs.jsx

```diff
@@ -18,6 +18,7 @@ let Tabs = React.createClass({
     initialSelectedIndex: React.PropTypes.number,
     inkBarStyle: React.PropTypes.object,
     tabItemContainerStyle: React.PropTypes.object,
+    tabTemplate: React.PropTypes.func,
```

Comment:
Using `React.PropTypes.node`, I get the following warning when I use the Tabs component:
`Invalid prop 'tabTemplate' supplied to 'Tabs', expected a ReactNode.`

Almost the same for `React.PropTypes.element`.

The reason is that we need to pass a javascript class to `tabTemplate` property instead of a `ReactElement`. Example:

```
import MyCustomTabTemplate from "components/MyCustomTabTemplate";

...

<Tabs tabTemplate={MyCustomTabTemplate}>
    ...
</Tabs>
```